bin/btc-server/src/database/error.rs

```diff
@@ -29,6 +29,10 @@ pub enum Error {
     OutputNotFound(usize),
     #[error("hash engine error {0}")]
     HashEngine(#[from] std::io::Error),
+    #[error("wrong version number {0}")]
+    InvalidVersion(u32),
```

Comment:
I'd be a bit more specific here.
```suggestion
    #[error("Invalid UTXO version number {0}")]
    InvalidUTXOVersion(u32),
```

Reply:
fixed